lib/pages/settings_dashboard/settings_profile/settings_profile_view.dart

```diff
@@ -80,51 +80,58 @@ class SettingsProfileView extends StatelessWidget {
             ): SlotLayout.from(
               key: settingsProfileViewMobileKey,
               builder: (_) {
-                return SettingsProfileViewMobile(
-                  client: controller.client,
-                  settingsProfileUIState: controller.settingsProfileUIState,
-                  onAvatarTap: () => controller.setAvatarAction(),
-                  settingsProfileOptions: ListView.separated(
-                    shrinkWrap: true,
-                    physics: const NeverScrollableScrollPhysics(),
-                    itemBuilder: (context, index) {
-                      return SettingsProfileItemBuilder(
-                        settingsProfileEnum:
-                            controller.getListProfileMobile[index],
-                        title: controller.getListProfileMobile[index]
-                            .getTitle(context),
-                        settingsProfileUIState:
-                            controller.settingsProfileUIState,
-                        settingsProfilePresentation:
-                            SettingsProfilePresentation(
-                          settingsProfileType: controller
-                              .getListProfileMobile[index]
-                              .getSettingsProfileType(),
-                        ),
-                        suffixIcon: controller.getListProfileMobile[index]
-                            .getTrailingIcon(),
-                        leadingIcon: controller.getListProfileMobile[index]
-                            .getLeadingIcon(),
-                        focusNode: controller.getFocusNode(
-                          controller.getListProfileMobile[index],
-                        ),
-                        textEditingController: controller.getController(
-                          controller.getListProfileMobile[index],
-                        ),
-                        onChange: (_, settingsProfileEnum) {
-                          controller
-                              .handleTextEditOnChange(settingsProfileEnum);
+                return ValueListenableBuilder(
+                  valueListenable: controller.settingsProfileUIState,
+                  builder: (context, _, __) {
```

Comment:
you don't use value to rebuild, so why need ValueListenableBuilder

Reply:
Sure, I'll remove it